File: Niveau 1/2 – Répétitions d'instructions/10 - Le Grand Événement.py
from robot import *
haut()
for i in range(10):
   for j in range(8):
      if i % 2 == 0:
         haut()
      else:
         bas()
   if i != 9:
      droite()
   else:
      bas()
for i in range(9):
   gauche()

File: Niveau 1/2 – Répétitions d'instructions/10 - Le Grand Événement.cpp
#include "robot.h"
using namespace std;

int main()
{
  droite();
  for (int i = 0; i < 5; i++)
  {
    for (int j = 0; j < 8; j++)
    {
      droite();
    }
    haut();
    for (int j = 0; j < 8; j++)
    {
      gauche();
    }
    if (i < 4)
    {
      haut();
    }
    else
    {
      gauche();
    }
  }
  for (int i = 0; i < 9; i++)
  {
    bas();
  }
}